namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-domain-events-prod/resources/hmpps-education-and-work-plan-api.tf

```diff
@@ -84,8 +84,8 @@ resource "aws_sns_topic_subscription" "education_and_work_plan_domain_events_sub
   endpoint  = module.education_and_work_plan_domain_events_queue.sqs_arn
   filter_policy = jsonencode({
     eventType = [
-      "ciag-induction.created",
```

Comment:
The `ciag-induction` events are no longer created and published by anything
(the CIAG API used to publish these events, but that API and codebase was decommissioned several months ago)